Sources/CartonCLI/Commands/TestRunners/BrowserTestRunner.swift

```diff
@@ -12,39 +12,165 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import AsyncHTTPClient
 import CartonHelpers
 import CartonKit
 import Foundation
+import NIOCore
+import NIOPosix
 import PackageModel
 import TSCBasic
+import WebDriverClient
 
 private enum Constants {
   static let entrypoint = Entrypoint(fileName: "test.js", sha256: testEntrypointSHA256)
 }
 
+enum BrowserTestRunnerError: Error, CustomStringConvertible {
+  case invalidRemoteURL(String)
+  case failedToFindDriver
```

Comment:
nit: let's make naming more explicit here
```suggestion
  case failedToFindWebDriver
```